checkstyle-custom/src/test/java/MethodParameterShouldBeOnePerLine.java

```diff
@@ -0,0 +1,50 @@
+import com.puppycrawl.tools.checkstyle.Checker;
```

Comment:
tests should be in the same base package as production code.

Don't ever use default package - it can lead to different side effects